Makefile

```diff
@@ -57,6 +56,11 @@ git_export: clean
 	# Windows needs a different name for the startup script - due to process creation
 	# (no fork/exec)
 	@cp "$(EXPORT_DIR)/scripts/pycam" "$(EXPORT_DIR)/scripts/pycam-loader.py"
+	# overwrite dynamic version detection with a hardcoded string and update deb changelog
+	cd "$(EXPORT_DIR)"; \
+		echo 'VERSION = "$(VERSION)"' >pycam/version.py; \
+		debchange --newversion "$$(echo "$(VERSION)" | tr - .)" "Upstream build"; \
+		debchange --release ""
```

Comment:
I like that you update the debian changelog version number, that's the most important part to update.  In my version i also add the oneline commit log to the changelog, which is sometimes handy:  53e8ee1428